### DISCUSSION TITLE
Support ignore white space changes on mobile too

### DISCUSSION THREAD

**Discussion by rart** [community/community#4283] [Mobile]
Please add support for the web option of ignore whitespace changes.

Would be great if there's a user setting to prefer that as default. This actually applies to both the web and mobile platforms.

**Comment by osdiab** [comment]
I agree - I use prettier on our JS codebase, that means code formatting stuff like white space is a non issue for us since prettier will always catch it - so white space is 99.9% of the time a distraction from the actual substance of our PRs, especially so since simple changes to nesting can cause large changes in whitespace. On desktop it should be defaultable, and on mobile it should also at least be an option, which is especially valuable since mobile has such constrained screen size.

**Reply by ishepherd** [reply]
Sooooooo much this. Please.
I only care what the code does (I'm not using [Whitespace](https://en.wikipedia.org/wiki/Whitespace_(programming_language)))

**Reply by zbyna** [reply]
It is needed feature! Commonly used in all major solutions for code compare.
